pages/content/posts/complete-interop-features/index.md

```diff
@@ -0,0 +1,69 @@
+
+## Opportunities created by the Interop initiative
+
+As a collective of writers dedicated to helping document the open web platform, [Open Web Docs (OWD)](https://openwebdocs.org/) can help in _closing some of these gaps_. Unlike writers who are paid by browser makers needing to justify their work, our backers, individuals or companies, gave us the more generic mission to improve the open web documentation. We focus on much-needed activities that do not necessarily bring the most engagement at first, though improving in the middle term situation significantly. We still need to focus on the most valuable activities, but anybody's internal projects do not dictate our priorities.
+
+For several years now, browser makers have sat together at the end of each year to discuss the pain points of web developers and define a set of features that needs more love to increase their functionality or to fix bugs and incoherence so that web devs can use them reliably. They define areas to focus on as part of the [Interop](https://web.dev/interop-2023/) initiative.
+
+OWD, and its partners, think these projects are good opportunities to revisit the relevant area of the MDN Web Docs documentation. The idea is to have the docs in good shape for each set of features made interoperable over the year. Most of the time, this leads to minor changes to the docs. MDN writers tried to describe the interoperable behaviour, such as only mentioning browsers when their implementations diverged.
+
+One of the Interop focus area this year, [interop-2023-forms](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-forms%20or%20label%3Ainterop-2023-forms), is about making HTML forms interoperable. It was an great opportunity to fill part of the MDN gaps. We decided to do the following:
+
+1. Document all interoperable APIs unrelated to HTML and SVG.
+2. Document all interoperable HTML APIs not part of the DOM (that is, everything in the HTML spec that is not an `HTML*Element` property or method).
+3. Finally, document all missing HTML*Element features related to `<form>` (For example, `HTMLSelectElement`, `HTMLInputElement`, or `HTMLButtonElement` properties and methods).
+
+The two first points let us detect any newly interoperable property or method and document it if this is the first time anyone has done it. We already found some after the launch of Safari 16.4 (like [`WritableStreamDefaultController.signal`](https://developer.mozilla.org/en-US/docs/Web/API/WritableStreamDefaultController/signal)). These two initial steps will enable us to maintain the completeness of our API reference for interoperable features once we finish documenting the backlog.
+
+The third step will give us a complete reference for the Forms APIs. It will be synchronized with this year's Interop initiative, where all three engines will be interoperable.
```

Comment:
```suggestion
The third step will give us a complete reference for the Forms APIs. It will be synchronized with this year's Interop initiative, where all three engines will be interoperable, fully supporting the Forms APIs.
```

Reply:
my edit may be redundant, but i just wanted to remind readers what interop means.